src/muya/lib/contentState/paragraphCtrl.js

```diff
@@ -18,6 +18,9 @@ const getCurrentLevel = type => {
 const paragraphCtrl = ContentState => {
   ContentState.prototype.selectionChange = function (cursor) {
     const { start, end } = cursor || selection.getCursorRange()
+    if (!start || !end) {
```

Comment:
I think this maybe also has exceptions, because the `selectionChange` expect to return a object like this bellow:

```js
  return {
      start,
      end,
      affiliation,
      cursorCoords
    }
```

Reply:
OK, I'll throw an exception and we should fix this in the upcoming muya implementation.